src/main/java/org/terasology/dynamicCities/sites/SiteFacetProvider.java

```diff
@@ -108,5 +102,8 @@ public void setConfiguration(Component configuration) {
 
         @Range(label = "Minimum distance between towns", min = 10, max = 1000, increment = 10, precision = 1)
         private int minDistance = 128;
+
+        @Range(label = "Minimum distance from spawn", min = 0, max = 1000, increment = 10, precision = 1)
+        private int minSpawnGap = 200;
```

Comment:
The configuration properties currently cannot differ from the default value (any change is lost when actually generating or re-loading a world). Just to let you know...

Reply:
That explains some weird behavior I was experiencing while testing (the setting didn't seem to change anything), I just thought it was my game acting weird. Thanks for telling me!